backend/src/test/java/ca/gov/dtsstn/cdcp/api/web/v1/controller/UsersControllerIT.java

```diff
@@ -94,7 +98,7 @@ void testGetUserById_NotFound() throws Exception {
 	@DisplayName("Test unauthenticated GET /api/v1/users/{id}")
 	void testGetUserById_Unauthorized() throws Exception {
 		mockMvc.perform(get("/api/v1/users/00000000-0000-0000-0000-000000000000"))
-			.andDo(print())
+			//.andDo(print())
```

Comment:
Why was this commented out?

Reply:
This commit was done while testing was still in progress, it seemed more convenient to reduce the output while I was investigating a failing test. They have been restored in commit 2cebd2d6f0685660fa512c66c4a5da16869070e1.